File: plugin/src/main/java/com/jonathanpotts/blockmaps/models/ServerModel.java
package com.jonathanpotts.blockmaps.models;

import java.util.List;

/**
 * Stores data used to process a server.
 */
public class ServerModel {
  /**
   * Message of the day on the server.
   */
  public String motd;

  /**
   * Worlds on the server.
   */
  public List<WorldModel> worlds;
}

File: src/main/kotlin/io/github/jonathanpotts/cartographytable/models/ServerModel.kt
package io.github.jonathanpotts.cartographytable.models

import kotlinx.serialization.Serializable

/**
 * Stores data used to process a server.
 *
 * @property motd Message of the day on the server.
 * @property worlds Worlds on the server.
 */
@Serializable
data class ServerModel(
    var motd: String = "",
    var worlds: MutableList<WorldModel> = mutableListOf()
)
